src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -212,14 +234,15 @@ def get_profile(
                 f"because {type(exc).__name__}'{exc}'.",
                 ConfigNotFoundWarning,
             )
-        finally:
-            return cfg_profile
+        
+        return cfg_profile
 
     def load(
         self,
         profile_name: Optional[str] = None,
         profile_type: Optional[str] = None,
         check_missing_props: bool = True,
+        validate_schema: Optional[bool] = True,
```

Comment:
should we add the parameter list to the description below?
I think these are self-descriptive parameters, so maybe there is no need to do this.